pkg/route/api/v1/conversion.go

```diff
@@ -13,6 +13,9 @@ func init() {
 			obj.To.Kind = "Service"
 		},
 		func(obj *TLSConfig) {
+			if len(obj.Termination) == 0 {
+				obj.Termination = TLSTerminationEdge
```

Comment:
sure, I'm down with that.  But since the router knows to DTRT I don't think it's that big of a deal.  If we really want to make it look like an unsecure route I'd say we unset the irrelevant data like you suggest.